Add Request a Priority Review page

cvalarida opened the pull request:
![image](https://user-images.githubusercontent.com/12970166/37003205-2273783e-2081-11e8-8854-a94ab99076b6.png)

Some outstanding questions:
- Should we use `h3`s?
  - @1Copenut advises us to go in order, h1 -> h2 -> h3, etc. for accessibility reasons, but it looks kinda gross
  - Also, it looks like a lot of other static content pages skip heading levels, so while this isn't great like it is now, it _is_ consistent
- Should the "By mail" and "By fax" be headers or just bold?
  - I'm leaning toward just bold
- Is the space between the "By mail" and "By fax" headers and content okay?

@bethpotts Gina said you wanted to review the content again; feel free to add any comments here if you're ready with them.

Commit:
Add Request a Priority Review page

gnakm commented:
Could we wrap the blue box to only include from "One of these must be true..." and the end of "Notes:...."

Commit:
Blue box

cvalarida commented:
@gnakm Done!
![image](https://user-images.githubusercontent.com/12970166/37004659-046c9cee-2086-11e8-96d8-75b67a374933.png)

bethpotts commented:
@cvalarida @gnakm 
1) To answer one of the above questions, we use #### for the subhead for "By mail" and "By fax"

2) Gina, your request to make the blue box smaller actually makes it less consistent with all our other static pages, so I'd recommend reverting that. (We always have a question at the top of the blue box.)

3) Here's the code for the address block:
<p class="va-address-block">
Board of Veterans’ Appeals<br>
PO Box 27063<br>
Washington, DC 20038<br>
</p>

cehsu commented:
Good question about the accessibility consideration with the heading order, would it make sense to file a ticket to style those so that we can use them in the appropriate order?

cehsu commented on content/pages/disability-benefits/claims-appeal/request-a-priority-review.md:
```diff
@@ -0,0 +1,48 @@
+plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
+template: detail-page
+---
+<div itemscope itemtype ="http://schema.org/HowTo">
```
Is there an extra space before `=`?

cvalarida replied:
Copy pasta. Good eye. :female_detective:

cehsu commented on content/pages/disability-benefits/claims-appeal/request-a-priority-review.md:
```diff
@@ -0,0 +1,48 @@
+<div class="feature">
+
+**One of these must be true. You:**
+- Are 75 years or older, **or**
```
Should this be "75 years old or older"?

gnakm replied:
yes

cehsu commented on content/pages/disability-benefits/claims-appeal/request-a-priority-review.md:
```diff
@@ -0,0 +1,48 @@
+Send to: Board of Veterans’ Appeals PO Box 27063 Washington, DC 20038
+
+##### By fax
+Fax to: <a href="tel:+12024956803">(202) 495-6803</a>
```
We occasionally include the country code in fax numbers (e.g. `pension-management-center.md`), should we do so here?

cehsu reviewed (approved):
LGTM once @gnakm @bethpotts approve.

cvalarida commented:
@bethpotts I think the trouble I'm having with putting "By mail" and "By fax" in a header is the parent section isn't in a header ("You or your representative can send your written request in 1 of 2 ways:"), so this makes kind of a weird hierarchy both visually and for screen readers.

### How do I request to have my appeal Advanced on Docket?
h3
#### You’ll need to include all of this information
h4 instead of bold
#### You or your representative can send your written request in 1 of 2 ways
h4 instead of bold
##### By mail
h5
##### By fax
h5

Commit:
Address block and headers

gnakm commented:
@cvalarida The mail and fax should be styled like this: 
![image](https://user-images.githubusercontent.com/13420618/37046117-8dd187b0-2135-11e8-9828-d871c0e7c033.png)

https://www.vets.gov/health-care/apply/

Commit:
Remove extra space

Commit:
Format address properly

cvalarida commented:
@gnakm Heh, I was reviewing one of Claire's PRs and saw that formatting was a thing. One of these days, I'd love to make a list of all the styling we have.

I can have dreams, right? :stuck_out_tongue_winking_eye: 

After a few changes, this is what I've got:
![image](https://user-images.githubusercontent.com/12970166/37049688-85cc762e-2126-11e8-8f76-056f88a7e7c8.png)

The changes are as follows:
- I added the `h4`s as I described above
  - If we want to change that, that's fine by me; I just thought I'd show what I was talking about
  - I do think this both looks better and makes more sense for header navigation in screen readers
- The mailing address is formatted properly now
- Removed the `:` after `Fax to`
- Removed `Send to:`

Commit:
Add old

cvalarida commented on content/pages/disability-benefits/claims-appeal/request-a-priority-review.md:
```diff
@@ -0,0 +1,52 @@
+- Are in financial distress, **or**
+- Have other sufficient cause (a reason for needing your appeal reviewed faster)
+
+**Note:** If you are older than 75, your appeal will receive Advanced on Docket status automatically.
```
:thinking: So is it >= 75, or just == 75?

Previously, we say "75 or older", and here it's just "older than 75."

gnakm replied:
Could you change this to "75 or older"

cehsu commented:
Adding @bethpotts as a reviewer as she's made several recommendations.

bethpotts committed:
Proofread: request-a-priority-review

bethpotts committed:
Proofread: request-a-priority-review

gnakm reviewed (approved):
Just went over all the content changes for this with Beth. LGTM!
